### PR TITLE
Inventory: show equipments by types

### PR DESCRIPTION
**Pull request by Indigo744**
Hello again!

To be more similar to units page, here is a PR to **present equipments by types**:

image

...

image

No name are shown, since there is, for now, no equivalent between internal type (e.g. `greatSword`) and full type name (e.g. `Great Sword`). Also it would have to be translated and all. So to keep it simple, only picture type is shown.

The async aspect was kept, but with types, the equipments are loaded by batch of same type (instead of batch of 20).

The materia listing is not modified, as the data currently doesn't allow us to discriminate between them (they all have `materia` as type). It would have been cool to sort by type as well (active, passive, etc...) like in game.

### CODE REVIEW

**Comment by lyrgard** [comment]
I like it ! 

For the materia, I hesitate. First, they are sorted alphabetically, to help input them, and in game, I never use the materia category, as they fell very lacking in pertinence most of the time... I mean, they could create a killer category, for instance, that would be usefull

File: static/inventory.js [hunk 1]
```diff
@@ -116,59 +118,94 @@ function showSettings() {
...
+var displayItems = function(items, byType = false) {
     var resultDiv = $("#results");
     resultDiv.empty();
     displayId++;
-    displayItemsAsync(items, 0, resultDiv, displayId);
+    if (byType) displayItemsByTypeAsync(items, 0, resultDiv, displayId);
+    else displayItemsAsync(items, 0, resultDiv, displayId);
+};
+
+function displayItemsByTypeAsync(items, start, div, id, currentItemType = null) {
```
Comment (lyrgard) [inline static/inventory.js]: You probably don't need to pass currentItemType. You can just do 

`var currentItemType = items[start].type`

no ?

Comment (Indigo744) [reply static/inventory.js]: Indeed! It's simpler.

**Comment by Indigo744** [comment]
Yes, the category are weird in game.

Maybe we can have our own useful categories?
The issue would be materia that falls into multiple categories (e.g. ATK passive + Killer...).

**Comment by Indigo744** [comment]
Note : I'm adding the jump list, please do not merge :wink:

**Comment by lyrgard** [comment]
I saw you comment just as I was going to merge it XD We're safe !

**Comment by Indigo744** [comment]
Done! It looks like this:
image

**Comment by lyrgard** [comment]
Merged an released ! Thanks !

**Pull request merged**